packages/ckeditor5-core/src/editor/editor.js

```diff
@@ -182,7 +182,8 @@ export default class Editor {
 		 * @readonly
 		 * @member {module:engine/controller/editingcontroller~EditingController}
 		 */
-		this.editing = new EditingController( this.model, stylesProcessor );
+		const sourceElementRoot = config.sourceElementRoot ? config.sourceElementRoot : document; // eslint-disable-line no-undef
+		this.editing = new EditingController( this.model, stylesProcessor, sourceElementRoot );
```

Comment:
I think this assignment needs to stay with the jsdoc comment above. Because this class isn't using class fields, the "field" comment goes with the first assignment in the constructor.